runtime/compiler/runtime/JITServerAOTCache.hpp

```diff
@@ -421,4 +455,23 @@ class JITServerAOTCacheMap
    };
 
 
+struct JITServerAOTCacheVersion
+   {
+   uint32_t snapshotVersion;
+   uint32_t jitserverVersionMajor;
```

Comment:
If this is the same thing as the JITServer protocol version defined in `CommunicationStream.hpp`, maybe we should store it the same way here? (as a single `uint32_t`)